crates/sui/src/client_commands.rs

```diff
@@ -1174,21 +1174,25 @@ impl WalletContext {
             .read_api()
             .get_objects_owned_by_address(address)
             .await?;
-
+        let o_ref_clone = object_refs.clone();
         // TODO: We should ideally fetch the objects from local cache
-        // TODO: replace with multi-get
         let mut values_objects = Vec::new();
-        for oref in object_refs {
-            let response = client
-                .read_api()
-                .get_object_with_options(oref.object_id, SuiObjectDataOptions::full_content())
-                .await?;
+        let oref_ids: Vec<ObjectID> = object_refs.into_iter().map(|oref| oref.object_id).collect();
+
+        let responses = client
+            .read_api()
+            .multi_get_object_with_options(oref_ids, SuiObjectDataOptions::full_content())
```

Comment:
I wanted to say this is a life changer, right now the CLI fails when the account holds more 20 objects due to rate-limiting. Thanks for improving this!!